charts/kcp/templates/etcd.yaml

```diff
@@ -153,11 +156,19 @@ spec:
               mountPath: /etc/etcd/tls/server
           resources:
             limits:
+              {{- if .Values.etcd.cpuLimit }}
```

Comment:
IMHO remove this as an option entirely

Reply:
This is copied from the downstream fork, so perhaps we should remove as a follow-up?

Reply:
+1, let's get it working, then consolidate & slim down